### DISCUSSION TITLE
Feedback from greater CISA SBOM Community

### DISCUSSION THREAD

**Discussion by idunbarh** [SBOM-Community/SBOM-Generation#47] [General]
Discussion to collect feedback on the phase 1 from the greater CISA SBOM Community.

### Keycloak
- [Workflow](https://github.com/CISA-SBOM-Community/SBOM-Generation/actions/runs/11348319620)
- [CycloneDX SBOM](https://github.com/CISA-SBOM-Community/SBOM-Generation/actions/runs/11348319620/artifacts/2058543214)
- [SPDX SBOM](https://github.com/CISA-SBOM-Community/SBOM-Generation/actions/runs/11348319620/artifacts/2058543141)

### Python App and Container
- [Workflow](https://github.com/CISA-SBOM-Community/SBOM-Generation/actions/runs/11348319619)
- [All SBOMs](https://github.com/CISA-SBOM-Community/SBOM-Generation/actions/runs/11348319619/artifacts/2058523267)

**Comment by anthonyharrison** [comment]
I can't see examples of the SBOMs which are being generated. It would be good to see what 'good' looks like.

I also note that there is a template 'SBOM'. It has defined the CycloneDX lifecycle as Build. I though the purpose of this activity was to produce 'Source' SBOMs? If multiple types of SBOMs are being produced (and why not?), I think it would be good to have these as separate examples to show that it is clear what additional information gets added at each stage of the SBOM lifecycle.

**Reply by idunbarh** [reply]
Hi @anthonyharrison! I added the following links to the top comment on discussion.

### Keycloak
- [Workflow](https://github.com/CISA-SBOM-Community/SBOM-Generation/actions/runs/11348319620)
- [CycloneDX SBOM](https://github.com/CISA-SBOM-Community/SBOM-Generation/actions/runs/11348319620/artifacts/2058543214)
- [SPDX SBOM](https://github.com/CISA-SBOM-Community/SBOM-Generation/actions/runs/11348319620/artifacts/2058543141)

### Python App and Container
- [Workflow](https://github.com/CISA-SBOM-Community/SBOM-Generation/actions/runs/11348319619)
- [All SBOMs](https://github.com/CISA-SBOM-Community/SBOM-Generation/actions/runs/11348319619/artifacts/2058523267)

**Reply by anthonyharrison** [reply]
Thanks @idunbarh . Lots of data to play with...

Can already see inconsistencies between the two formats. I suppose what is missing is what the 'ground' truth is so we can then determine which one is right (if at all).

I think it would also be useful to include the raw SBOMs (before the enrichment phase) so that a comparison can be made against the enriched version. Many organisations are just relying on a tool generated SBOM without enrichment.

**Reply by vpetersson** [reply]
@anthonyharrison As was mentioned in yesterday's SBOM meeting, all steps are included in the final artifacts. Thus it's possible to diff each phase as @tiegz illustrated.
